src/me/sirfaizdat/prison/core/Config.java

```diff
@@ -78,6 +78,18 @@ private void loadAll() {
             enableMultiworld = c.getBoolean("multiworld");
             fireworksOnRankup = c.getBoolean("fireworks-on-rankup");
             flamesOnAutosmelt = c.getBoolean("flames-on-autosmelt");
+            boolean configerr = false;
```

Comment:
Slight nitpick here, but why is this variable named `configerr`? Perhaps a more descriptive/clear name could be used, or maybe I am misunderstanding something.